src/main/java/uk/gov/hmcts/darts/common/client/AbstractRestTemplateClient.java

```diff
@@ -42,6 +45,15 @@ public <T> void streamFileWithMetaData(MultipartFile multipartFile, T metadata,
         headers.setContentType(MediaType.MULTIPART_FORM_DATA);
 
         try {
+            String metaDataString = "Unknown request payload";
+            try {
+                ObjectMapper mapper = ServiceConfig.getServiceObjectMapper();
+                metaDataString = mapper.writeValueAsString(metadata);
+            } catch (JsonProcessingException jsonProcessingException) {
+                log.warn("Problem marshalling meta data payload", jsonProcessingException);
```

Comment:
should be log.error

Reply:
yeah its a warn as processing will continue